config/locales/en.yml

```diff
@@ -33,6 +33,8 @@ en:
               too_long: Question text must be %{count} characters or less
         user:
           attributes:
+            name:
+              blank: You must enter a name for editor or super admin users
             organisation_id:
               blank: Select the user’s organisation
```

Comment:
@hannahkc as well as making it so that the user has to have a name set unless they have a trial account, I also made it an error to unset their name if they already have one set (we do the same thing for the organisation, so I thought it would be best to be consistent). Do you think we need a different error message for that scenario?

Also, there's a bit of inconsistency between the different error messages for this page, do we want to do anything about that?

<img width="669" alt="Screenshot 2023-10-05 at 08 46 40" src="https://github.com/alphagov/forms-admin/assets/503614/c7e2fd09-9983-4a28-86aa-8be36988c39f">

Reply:
Going to resolve this in a separate PR.